bigchaindb/common/transaction.py

```diff
@@ -647,6 +498,8 @@ def create(cls, owners_before, owners_after, metadata=None, asset=None):
             raise ValueError('`owners_before` list cannot be empty')
         if len(owners_after) == 0:
             raise ValueError('`owners_after` list cannot be empty')
+        if not (asset is None or isinstance(asset, dict)):
```

Comment:
Not sure if this check is needed since this check will be performed when calling `cls(...)`